e2e_tests/tests/requirements.txt

```diff
@@ -5,7 +5,7 @@ pexpect
 torch==1.4
 torchvision==0.5.0
 tensorflow==1.15.4
-numpy
+numpy==1.16.2
```

Comment:
Looking at the warnings you listed in the PR message and the [numpy version history](https://pypi.org/project/numpy/#history), wouldn't `0.18.5` work?

Also, whatever you put here, please put a comment here explaining that this is to "manually resolve a numpy version conflict between tf 1.15.4 and determined that is not well-resolved by pypi" or something.  Otherwise it can be awfully hard to figure out why some things got pinned later down the road, which is important to determine when it is safe to unpin things later.

Reply:
I downgraded to 0.16.x because of the numpy.ufunc issue: https://stackoverflow.com/questions/60323366/valueerror-numpy-ufunc-size-changed-may-indicate-binary-incompatibility-expec. However, I didn't try most versions above 0.16.x so it's quite possible that there are other numpy versions that work. In any case, I wanted to document my experience to engineering.